### ISSUE TITLE
[Unhandled exception] AttributeError

### ISSUE THREAD

**Issue by datamaskinen** [OpenCobolIDE/OpenCobolIDE#385]
### Description

An unhandled exception has occured:

  File "/usr/lib/python3/dist-packages/open_cobol_ide/controllers/cobol.py", line 249, in clean
    path = self.app.edit.current_editor.file.path

AttributeError: 'NoneType' object has no attribute 'file'

Would you like to send a bug report to the development team?

### Traceback

```
  File "/usr/lib/python3/dist-packages/open_cobol_ide/controllers/cobol.py", line 249, in clean
    path = self.app.edit.current_editor.file.path

AttributeError: 'NoneType' object has no attribute 'file'
```

### System information

- Operating System: Ubuntu
- OpenCobolIDE: 4.7.4
- GnuCOBOL: cobc (OpenCOBOL) 1.1.0
- Python: 3.5.2+ (64bits)
- Qt: 5.6.1
- PyQt: 5.7
- pyqode.core: 2.10.1
- pyqode.cobol: 2.10.1
- pyqode.qt: 2.10.0
- pygments: 2.1.3
- QDarkStyle: 2.2

Application log: https://gist.github.com/dbe91e5f48dbfdaa77f345fc7700cea6

**Comment by datamaskinen** [comment]
when i start opencobolide and try to open my helloworld project i get this dialog
'Failed to open path: connect() failed between triggered(bool) and to_lower()'

Then i push ok and it starts up the ide....
I den push compile or any of the 4 icons there  at the top and gets this dialog
'An unhandled exception has occured:

  File "/usr/lib/python3/dist-packages/open_cobol_ide/controllers/cobol.py", line 214, in compile
    path = self.app.edit.current_editor.file.path

AttributeError: 'NoneType' object has no attribute 'file'

Would you like to send a bug report to the development team?'


The about .... log in DEBUG mode show this:
10:23:39:101::INFO::8857::open_cobol_ide::version: 4.7.4
10:23:39:102::INFO::8857::open_cobol_ide.app::files to open: []
10:23:40:459::DEBUG::8857::open_cobol_ide.compilers::removing executable test if still exists...
10:23:40:459::DEBUG::8857::open_cobol_ide.compilers::check compiler
10:23:40:460::DEBUG::8857::open_cobol_ide.compilers::PATH=:/home/marlin/bin:/home/marlin/.local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin
10:23:40:460::DEBUG::8857::open_cobol_ide.compilers::command: cobc -v -x /tmp/test.cbl
10:23:40:461::DEBUG::8857::open_cobol_ide.compilers::working directory: /tmp
10:23:40:461::DEBUG::8857::open_cobol_ide.compilers::environment: ['PWD=/home/marlin', 'LC_MEASUREMENT=sv_SE.UTF-8', 'GTK_RC_FILES=/etc/gtk/gtkrc:/home/marlin/.gtkrc:/home/marlin/.config/gtkrc', 'SSH_AUTH_SOCK=/tmp/ssh-2tz8jgITomqW/agent.1439', 'LANG=en_US.UTF-8', 'SHELL=/bin/bash', 'USER=marlin', 'XDG_RUNTIME_DIR=/run/user/1000', 'DBUS_SESSION_BUS_ADDRESS=unix:abstract=/tmp/dbus-F296bi0umU,guid=07d6b08f5b4c05eaf500ed31584ad254', 'PAM_KWALLET_LOGIN=/tmp/kwallet_marlin.socket', 'GS_LIB=/home/marlin/.fonts', 'XCURSOR_THEME=breeze_cursors', 'LC_ADDRESS=sv_SE.UTF-8', 'DESKTOP_STARTUP_ID=slaptop;1481793814;58054;1632_TIME280264707', 'QT_LINUX_ACCESSIBILITY_ALWAYS_ON=1', 'XDG_SEAT=seat0', 'XDG_SESSION_PATH=/org/freedesktop/DisplayManager/Session1', 'DEFAULTS_PATH=/usr/share/gconf//usr/share/xsessions/plasma.default.path', 'XDG_SEAT_PATH=/org/freedesktop/DisplayManager/Seat0', 'MANDATORY_PATH=/usr/share/gconf//usr/share/xsessions/plasma.mandatory.path', 'PATH=:/home/marlin/bin:/home/marlin/.local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin', 'KDE_MULTIHEAD=false', 'KDE_FULL_SESSION=true', 'XDG_SESSION_ID=3', 'XDG_SESSION_TYPE=x11', 'LC_PAPER=sv_SE.UTF-8', 'XDG_SESSION_CLASS=user', 'XDG_CONFIG_DIRS=/etc/xdg/xdg-/usr/share/xsessions/plasma:/etc/xdg:/usr/share/kubuntu-default-settings/kf5-settings', 'QT_ACCESSIBILITY=1', 'OCIDE_EXTLIBS_PATH=/usr/lib/python3/dist-packages/open_cobol_ide/extlibs', 'XDG_DATA_DIRS=/usr/share//usr/share/xsessions/plasma:/usr/local/share/:/usr/share/:/var/lib/snapd/desktop', 'PAM_KWALLET5_LOGIN=/tmp/kwallet5_marlin.socket', 'DESKTOP_SESSION=/usr/share/xsessions/plasma', 'SHLVL=0', 'XDG_CURRENT_DESKTOP=KDE', 'KDE_SESSION_VERSION=5', 'LC_NAME=sv_SE.UTF-8', 'LC_TELEPHONE=sv_SE.UTF-8', 'LC_TIME=sv_SE.UTF-8', 'GTK2_RC_FILES=/etc/gtk-2.0/gtkrc:/home/marlin/.gtkrc-2.0:/home/marlin/.config/gtkrc-2.0', 'XDG_VTNR=7', 'KDE_SESSION_UID=1000', 'SSH_AGENT_PID=1501', 'QT_API=pyqt5', 'DISPLAY=:0', 'LC_IDENTIFICATION=sv_SE.UTF-8', 'LOGNAME=marlin', 'QT_AUTO_SCREEN_SCALE_FACTOR=0', 'SESSION_MANAGER=local/slaptop:@/tmp/.ICE-unix/1569,unix/slaptop:/tmp/.ICE-unix/1569', 'LC_MONETARY=sv_SE.UTF-8', 'HOME=/home/marlin', 'LC_NUMERIC=sv_SE.UTF-8', 'XDG_SESSION_DESKTOP=KDE', 'XAUTHORITY=/tmp/xauth-1000-_0']
10:23:42:800::DEBUG::8857::open_cobol_ide.compilers::output: 'preprocessing /tmp/test.cbl into /tmp/cob8874_0.cob\ntranslating /tmp/cob8874_0.cob into /tmp/cob8874_0.c\ngcc -pipe -c  -Wno-unused -fsigned-char -Wno-pointer-sign  -o /tmp/cob8874_0.o /tmp/cob8874_0.c\ngcc -pipe  -Wl,--export-dynamic -o test /tmp/cob8874_0.o  -L/usr/lib -lcob -lm -lgmp -lncurses -ldb -ldl\n'
10:23:42:801::DEBUG::8857::open_cobol_ide.compilers::exit code: 0
10:23:42:802::DEBUG::8857::open_cobol_ide.compilers::removing executable test if still exists...
10:23:42:803::DEBUG::8857::open_cobol_ide.compilers::PATH=:/home/marlin/bin:/home/marlin/.local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin
10:23:42:803::DEBUG::8857::open_cobol_ide.compilers::command: cobc -v /tmp/test.cbl
10:23:42:803::DEBUG::8857::open_cobol_ide.compilers::working directory: /tmp
10:23:42:803::DEBUG::8857::open_cobol_ide.compilers::environment: ['PWD=/home/marlin', 'LC_MEASUREMENT=sv_SE.UTF-8', 'GTK_RC_FILES=/etc/gtk/gtkrc:/home/marlin/.gtkrc:/home/marlin/.config/gtkrc', 'SSH_AUTH_SOCK=/tmp/ssh-2tz8jgITomqW/agent.1439', 'LANG=en_US.UTF-8', 'SHELL=/bin/bash', 'USER=marlin', 'XDG_RUNTIME_DIR=/run/user/1000', 'DBUS_SESSION_BUS_ADDRESS=unix:abstract=/tmp/dbus-F296bi0umU,guid=07d6b08f5b4c05eaf500ed31584ad254', 'PAM_KWALLET_LOGIN=/tmp/kwallet_marlin.socket', 'GS_LIB=/home/marlin/.fonts', 'XCURSOR_THEME=breeze_cursors', 'LC_ADDRESS=sv_SE.UTF-8', 'DESKTOP_STARTUP_ID=slaptop;1481793814;58054;1632_TIME280264707', 'QT_LINUX_ACCESSIBILITY_ALWAYS_ON=1', 'XDG_SEAT=seat0', 'XDG_SESSION_PATH=/org/freedesktop/DisplayManager/Session1', 'DEFAULTS_PATH=/usr/share/gconf//usr/share/xsessions/plasma.default.path', 'XDG_SEAT_PATH=/org/freedesktop/DisplayManager/Seat0', 'MANDATORY_PATH=/usr/share/gconf//usr/share/xsessions/plasma.mandatory.path', 'PATH=:/home/marlin/bin:/home/marlin/.local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin', 'KDE_MULTIHEAD=false', 'KDE_FULL_SESSION=true', 'XDG_SESSION_ID=3', 'XDG_SESSION_TYPE=x11', 'LC_PAPER=sv_SE.UTF-8', 'XDG_SESSION_CLASS=user', 'XDG_CONFIG_DIRS=/etc/xdg/xdg-/usr/share/xsessions/plasma:/etc/xdg:/usr/share/kubuntu-default-settings/kf5-settings', 'QT_ACCESSIBILITY=1', 'OCIDE_EXTLIBS_PATH=/usr/lib/python3/dist-packages/open_cobol_ide/extlibs', 'XDG_DATA_DIRS=/usr/share//usr/share/xsessions/plasma:/usr/local/share/:/usr/share/:/var/lib/snapd/desktop', 'PAM_KWALLET5_LOGIN=/tmp/kwallet5_marlin.socket', 'DESKTOP_SESSION=/usr/share/xsessions/plasma', 'SHLVL=0', 'XDG_CURRENT_DESKTOP=KDE', 'KDE_SESSION_VERSION=5', 'LC_NAME=sv_SE.UTF-8', 'LC_TELEPHONE=sv_SE.UTF-8', 'LC_TIME=sv_SE.UTF-8', 'GTK2_RC_FILES=/etc/gtk-2.0/gtkrc:/home/marlin/.gtkrc-2.0:/home/marlin/.config/gtkrc-2.0', 'XDG_VTNR=7', 'KDE_SESSION_UID=1000', 'SSH_AGENT_PID=1501', 'QT_API=pyqt5', 'DISPLAY=:0', 'LC_IDENTIFICATION=sv_SE.UTF-8', 'LOGNAME=marlin', 'QT_AUTO_SCREEN_SCALE_FACTOR=0', 'SESSION_MANAGER=local/slaptop:@/tmp/.ICE-unix/1569,unix/slaptop:/tmp/.ICE-unix/1569', 'LC_MONETARY=sv_SE.UTF-8', 'HOME=/home/marlin', 'LC_NUMERIC=sv_SE.UTF-8', 'XDG_SESSION_DESKTOP=KDE', 'XAUTHORITY=/tmp/xauth-1000-_0']
10:23:42:910::DEBUG::8857::open_cobol_ide.compilers::output: 'preprocessing /tmp/test.cbl into /tmp/cob8883_0.cob\ntranslating /tmp/cob8883_0.cob into /tmp/cob8883_0.c\ngcc -pipe  -Wno-unused -fsigned-char -Wno-pointer-sign  -shared  -fPIC -DPIC -Wl,--export-dynamic -o test.so /tmp/cob8883_0.c -L/usr/lib -lcob -lm -lgmp -lncurses -ldb -ldl\n'
10:23:42:911::DEBUG::8857::open_cobol_ide.compilers::exit code: 0
10:23:42:912::INFO::8857::open_cobol_ide.compilers::GnuCOBOL compiler check: success (1/1)
10:23:42:912::INFO::8857::open_cobol_ide.compilers::Executable extension: 
10:23:42:912::INFO::8857::open_cobol_ide.compilers::Module extension: .so
10:23:42:914::DEBUG::8857::open_cobol_ide.compilers::getting cobc version: cobc --version
10:23:42:914::DEBUG::8857::open_cobol_ide.compilers::PATH=:/home/marlin/bin:/home/marlin/.local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin
10:23:42:915::DEBUG::8857::open_cobol_ide.compilers::command: cobc --version
10:23:42:915::DEBUG::8857::open_cobol_ide.compilers::working directory: 
10:23:42:915::DEBUG::8857::open_cobol_ide.compilers::environment: ['PWD=/home/marlin', 'LC_MEASUREMENT=sv_SE.UTF-8', 'GTK_RC_FILES=/etc/gtk/gtkrc:/home/marlin/.gtkrc:/home/marlin/.config/gtkrc', 'SSH_AUTH_SOCK=/tmp/ssh-2tz8jgITomqW/agent.1439', 'LANG=en_US.UTF-8', 'SHELL=/bin/bash', 'USER=marlin', 'XDG_RUNTIME_DIR=/run/user/1000', 'DBUS_SESSION_BUS_ADDRESS=unix:abstract=/tmp/dbus-F296bi0umU,guid=07d6b08f5b4c05eaf500ed31584ad254', 'PAM_KWALLET_LOGIN=/tmp/kwallet_marlin.socket', 'GS_LIB=/home/marlin/.fonts', 'XCURSOR_THEME=breeze_cursors', 'LC_ADDRESS=sv_SE.UTF-8', 'DESKTOP_STARTUP_ID=slaptop;1481793814;58054;1632_TIME280264707', 'QT_LINUX_ACCESSIBILITY_ALWAYS_ON=1', 'XDG_SEAT=seat0', 'XDG_SESSION_PATH=/org/freedesktop/DisplayManager/Session1', 'DEFAULTS_PATH=/usr/share/gconf//usr/share/xsessions/plasma.default.path', 'XDG_SEAT_PATH=/org/freedesktop/DisplayManager/Seat0', 'MANDATORY_PATH=/usr/share/gconf//usr/share/xsessions/plasma.mandatory.path', 'PATH=:/home/marlin/bin:/home/marlin/.local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin', 'KDE_MULTIHEAD=false', 'KDE_FULL_SESSION=true', 'XDG_SESSION_ID=3', 'XDG_SESSION_TYPE=x11', 'LC_PAPER=sv_SE.UTF-8', 'XDG_SESSION_CLASS=user', 'XDG_CONFIG_DIRS=/etc/xdg/xdg-/usr/share/xsessions/plasma:/etc/xdg:/usr/share/kubuntu-default-settings/kf5-settings', 'QT_ACCESSIBILITY=1', 'OCIDE_EXTLIBS_PATH=/usr/lib/python3/dist-packages/open_cobol_ide/extlibs', 'XDG_DATA_DIRS=/usr/share//usr/share/xsessions/plasma:/usr/local/share/:/usr/share/:/var/lib/snapd/desktop', 'PAM_KWALLET5_LOGIN=/tmp/kwallet5_marlin.socket', 'DESKTOP_SESSION=/usr/share/xsessions/plasma', 'SHLVL=0', 'XDG_CURRENT_DESKTOP=KDE', 'KDE_SESSION_VERSION=5', 'LC_NAME=sv_SE.UTF-8', 'LC_TELEPHONE=sv_SE.UTF-8', 'LC_TIME=sv_SE.UTF-8', 'GTK2_RC_FILES=/etc/gtk-2.0/gtkrc:/home/marlin/.gtkrc-2.0:/home/marlin/.config/gtkrc-2.0', 'XDG_VTNR=7', 'KDE_SESSION_UID=1000', 'SSH_AGENT_PID=1501', 'QT_API=pyqt5', 'DISPLAY=:0', 'LC_IDENTIFICATION=sv_SE.UTF-8', 'LOGNAME=marlin', 'QT_AUTO_SCREEN_SCALE_FACTOR=0', 'SESSION_MANAGER=local/slaptop:@/tmp/.ICE-unix/1569,unix/slaptop:/tmp/.ICE-unix/1569', 'LC_MONETARY=sv_SE.UTF-8', 'HOME=/home/marlin', 'LC_NUMERIC=sv_SE.UTF-8', 'XDG_SESSION_DESKTOP=KDE', 'XAUTHORITY=/tmp/xauth-1000-_0']
10:23:42:922::DEBUG::8857::open_cobol_ide.compilers::output: 'cobc (OpenCOBOL) 1.1.0\nCopyright (C) 2001-2009 Keisuke Nishida / Roger While\nBuilt    Aug 04 2016 15:56:22\nPackaged Feb 06 2009 10:30:55 CET\n'
10:23:42:922::DEBUG::8857::open_cobol_ide.compilers::exit code: 0
10:23:42:922::DEBUG::8857::open_cobol_ide.compilers::parsing version line: cobc (OpenCOBOL) 1.1.0
Copyright (C) 2001-2009 Keisuke Nishida / Roger While
Built    Aug 04 2016 15:56:22
Packaged Feb 06 2009 10:30:55 CET

10:23:42:923::INFO::8857::open_cobol_ide.app::GnuCOBOL version: cobc (OpenCOBOL) 1.1.0
10:23:43:076::DEBUG::8857::open_cobol_ide.controllers.view::showing page <Page.HOME: 0>
10:23:43:209::ERROR::8857::pyqode.backend::Traceback (most recent call last):
10:23:43:209::ERROR::8857::pyqode.backend::  File "/usr/lib/python3/dist-packages/open_cobol_ide/extlibs/pyqode/core/backend/server.py", line 235, in <module>
10:23:43:209::ERROR::8857::pyqode.backend::    from pyqode.core import backend
10:23:43:209::ERROR::8857::pyqode.backend::ImportError: No module named 'pyqode'
10:23:44:408::ERROR::8857::pyqode.backend::Traceback (most recent call last):
10:23:44:408::ERROR::8857::pyqode.backend::  File "/usr/lib/python3/dist-packages/open_cobol_ide/extlibs/pyqode/core/backend/server.py", line 235, in <module>
10:23:44:408::ERROR::8857::pyqode.backend::    from pyqode.core import backend
10:23:44:408::ERROR::8857::pyqode.backend::ImportError: No module named 'pyqode'
10:25:06:550::DEBUG::8857::open_cobol_ide.controllers.view::showing page <Page.EDIT: 1>
10:25:08:080::CRITICAL::8857::qcrash._hooks::unhandled exception:
  File "/usr/lib/python3/dist-packages/open_cobol_ide/controllers/cobol.py", line 214, in compile
    path = self.app.edit.current_editor.file.path

AttributeError: 'NoneType' object has no attribute 'file'
10:25:08:081::CRITICAL::8857::open_cobol_ide.app::unhandled exception:
  File "/usr/lib/python3/dist-packages/open_cobol_ide/controllers/cobol.py", line 214, in compile
    path = self.app.edit.current_editor.file.path

AttributeError: 'NoneType' object has no attribute 'file'
10:26:02:761::CRITICAL::8857::qcrash._hooks::unhandled exception:
  File "/usr/lib/python3/dist-packages/open_cobol_ide/controllers/cobol.py", line 249, in clean
    path = self.app.edit.current_editor.file.path

AttributeError: 'NoneType' object has no attribute 'file'
10:26:02:763::CRITICAL::8857::open_cobol_ide.app::unhandled exception:
  File "/usr/lib/python3/dist-packages/open_cobol_ide/controllers/cobol.py", line 249, in clean
    path = self.app.edit.current_editor.file.path

AttributeError: 'NoneType' object has no attribute 'file'
10:26:07:730::CRITICAL::8857::qcrash._hooks::unhandled exception:
  File "/usr/lib/python3/dist-packages/open_cobol_ide/controllers/cobol.py", line 269, in rebuild
    self.clean()
  File "/usr/lib/python3/dist-packages/open_cobol_ide/controllers/cobol.py", line 249, in clean
    path = self.app.edit.current_editor.file.path

AttributeError: 'NoneType' object has no attribute 'file'
10:26:07:731::CRITICAL::8857::open_cobol_ide.app::unhandled exception:
  File "/usr/lib/python3/dist-packages/open_cobol_ide/controllers/cobol.py", line 269, in rebuild
    self.clean()
  File "/usr/lib/python3/dist-packages/open_cobol_ide/controllers/cobol.py", line 249, in clean
    path = self.app.edit.current_editor.file.path

AttributeError: 'NoneType' object has no attribute 'file'
10:26:12:011::CRITICAL::8857::qcrash._hooks::unhandled exception:
  File "/usr/lib/python3/dist-packages/open_cobol_ide/controllers/cobol.py", line 359, in run
    self.compile()
  File "/usr/lib/python3/dist-packages/open_cobol_ide/controllers/cobol.py", line 214, in compile
    path = self.app.edit.current_editor.file.path

AttributeError: 'NoneType' object has no attribute 'file'
10:26:12:011::CRITICAL::8857::open_cobol_ide.app::unhandled exception:
  File "/usr/lib/python3/dist-packages/open_cobol_ide/controllers/cobol.py", line 359, in run
    self.compile()
  File "/usr/lib/python3/dist-packages/open_cobol_ide/controllers/cobol.py", line 214, in compile
    path = self.app.edit.current_editor.file.path

AttributeError: 'NoneType' object has no attribute 'file'
10:27:34:610::DEBUG::8857::open_cobol_ide.compilers::getting cobc version: cobc --version
10:27:34:611::DEBUG::8857::open_cobol_ide.compilers::PATH=:/home/marlin/bin:/home/marlin/.local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin
10:27:34:611::DEBUG::8857::open_cobol_ide.compilers::command: cobc --version
10:27:34:611::DEBUG::8857::open_cobol_ide.compilers::working directory: 
10:27:34:612::DEBUG::8857::open_cobol_ide.compilers::environment: ['PWD=/home/marlin', 'LC_MEASUREMENT=sv_SE.UTF-8', 'GTK_RC_FILES=/etc/gtk/gtkrc:/home/marlin/.gtkrc:/home/marlin/.config/gtkrc', 'SSH_AUTH_SOCK=/tmp/ssh-2tz8jgITomqW/agent.1439', 'LANG=en_US.UTF-8', 'SHELL=/bin/bash', 'USER=marlin', 'XDG_RUNTIME_DIR=/run/user/1000', 'DBUS_SESSION_BUS_ADDRESS=unix:abstract=/tmp/dbus-F296bi0umU,guid=07d6b08f5b4c05eaf500ed31584ad254', 'PAM_KWALLET_LOGIN=/tmp/kwallet_marlin.socket', 'GS_LIB=/home/marlin/.fonts', 'XCURSOR_THEME=breeze_cursors', 'LC_ADDRESS=sv_SE.UTF-8', 'DESKTOP_STARTUP_ID=slaptop;1481793814;58054;1632_TIME280264707', 'QT_LINUX_ACCESSIBILITY_ALWAYS_ON=1', 'XDG_SEAT=seat0', 'XDG_SESSION_PATH=/org/freedesktop/DisplayManager/Session1', 'DEFAULTS_PATH=/usr/share/gconf//usr/share/xsessions/plasma.default.path', 'XDG_SEAT_PATH=/org/freedesktop/DisplayManager/Seat0', 'MANDATORY_PATH=/usr/share/gconf//usr/share/xsessions/plasma.mandatory.path', 'PATH=:/home/marlin/bin:/home/marlin/.local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin', 'KDE_MULTIHEAD=false', 'KDE_FULL_SESSION=true', 'XDG_SESSION_ID=3', 'XDG_SESSION_TYPE=x11', 'LC_PAPER=sv_SE.UTF-8', 'XDG_SESSION_CLASS=user', 'XDG_CONFIG_DIRS=/etc/xdg/xdg-/usr/share/xsessions/plasma:/etc/xdg:/usr/share/kubuntu-default-settings/kf5-settings', 'QT_ACCESSIBILITY=1', 'OCIDE_EXTLIBS_PATH=/usr/lib/python3/dist-packages/open_cobol_ide/extlibs', 'XDG_DATA_DIRS=/usr/share//usr/share/xsessions/plasma:/usr/local/share/:/usr/share/:/var/lib/snapd/desktop', 'PAM_KWALLET5_LOGIN=/tmp/kwallet5_marlin.socket', 'DESKTOP_SESSION=/usr/share/xsessions/plasma', 'SHLVL=0', 'XDG_CURRENT_DESKTOP=KDE', 'KDE_SESSION_VERSION=5', 'LC_NAME=sv_SE.UTF-8', 'LC_TELEPHONE=sv_SE.UTF-8', 'LC_TIME=sv_SE.UTF-8', 'GTK2_RC_FILES=/etc/gtk-2.0/gtkrc:/home/marlin/.gtkrc-2.0:/home/marlin/.config/gtkrc-2.0', 'XDG_VTNR=7', 'KDE_SESSION_UID=1000', 'SSH_AGENT_PID=1501', 'QT_API=pyqt5', 'DISPLAY=:0', 'LC_IDENTIFICATION=sv_SE.UTF-8', 'LOGNAME=marlin', 'QT_AUTO_SCREEN_SCALE_FACTOR=0', 'SESSION_MANAGER=local/slaptop:@/tmp/.ICE-unix/1569,unix/slaptop:/tmp/.ICE-unix/1569', 'LC_MONETARY=sv_SE.UTF-8', 'HOME=/home/marlin', 'LC_NUMERIC=sv_SE.UTF-8', 'XDG_SESSION_DESKTOP=KDE', 'XAUTHORITY=/tmp/xauth-1000-_0']
10:27:34:619::DEBUG::8857::open_cobol_ide.compilers::output: 'cobc (OpenCOBOL) 1.1.0\nCopyright (C) 2001-2009 Keisuke Nishida / Roger While\nBuilt    Aug 04 2016 15:56:22\nPackaged Feb 06 2009 10:30:55 CET\n'
10:27:34:620::DEBUG::8857::open_cobol_ide.compilers::exit code: 0
10:27:34:620::DEBUG::8857::open_cobol_ide.compilers::parsing version line: cobc (OpenCOBOL) 1.1.0
Copyright (C) 2001-2009 Keisuke Nishida / Roger While
Built    Aug 04 2016 15:56:22
Packaged Feb 06 2009 10:30:55 CET

**Comment by datamaskinen** [comment]
Sorry i forgot to tell my distro:
Linux slaptop 4.8.0-30-generic #32-Ubuntu SMP Fri Dec 2 03:43:27 UTC 2016 x86_64 x86_64 x86_64 GNU/Linux

**Comment by datamaskinen** [comment]
I have tried to install from pip and i have installed from deb both python3-opencobolide_4.7.3-1_all.deb and python3-opencobolide_4.7.4-1_all.deb .... the same message on all 3

**Comment by ColinDuquesnoy** [comment]
Known issue alreasy fixes in v4.7.5, see #379
